src/lib/Microsoft.Health.Events/Resources/EventResources.resx

```diff
@@ -0,0 +1,144 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file needed?

Reply:
Yes, I added this resource file to localize the strings that we expose to the customers, in this case I've added the custom exception messages.